clang/docs/ReleaseNotes.rst

```diff
@@ -585,10 +585,11 @@ Bug Fixes to C++ Support
 - Clang incorrectly considered a class with an anonymous union member to not be
   const-default-constructible even if a union member has a default member initializer.
   (#GH95854).
-- Fixed an assertion failure when evaluating an invalid expression in an array initializer. (#GH112140)
+- Fixed an assertion failure when evaluating an invalid expression in an array initializer (#GH112140)
```

Comment:
nit: Why drop the dot here?

Reply:
@zyn0217 oops, it looks like it was removed during the merge conflict resolution. I've reverted it — thanks.